### PR TITLE
KEYCLOAK-12473 Add possibility to specify length of event detail when…

### PR DESCRIPTION
**Pull request by vramik**
… storing to database

### CODE REVIEW

**Comment by vramik** [comment]
https://github.com/keycloak/keycloak/pull/6576

File: server_admin/topics/events/login.adoc [hunk 1]
```diff
@@ -104,5 +104,19 @@ that comes with your distribution and adding for example:
 </spi>
 ----
 
+You can also setup a maximum length of the Event detail stored in the database by editing `standalone.xml`, `standalone-ha.xml`, or
```
Comment (andymunro) [inline server_admin/topics/events/login.adoc]: Setup as a verb should be two words:  "Set up"

File: server_admin/topics/events/login.adoc [hunk 2]
```diff
@@ -104,5 +104,19 @@ that comes with your distribution and adding for example:
 </spi>
 ----
 
+You can also setup a maximum length of the Event detail stored in the database by editing `standalone.xml`, `standalone-ha.xml`, or 
+`domain.xml`. This can be useful in case some field (e.g. redirect_uri) is very long. There is an example how the setup can look like:
```
Comment (andymunro) [inline server_admin/topics/events/login.adoc]: Try to always follow "this" with a noun.  "This action"  Or make it "Limiting the event detail...."

Also, consider this wording: "Here is an example of defining the maximum length.:

**Review by andymunro — DISMISSED** [review]
Approved with the suggested changes.

**Review by stianst — DISMISSED** [review]

**Review by pdrozd — DISMISSED** [review]

**Review by hmlnarik — DISMISSED** [review]

**Comment by hmlnarik** [comment]
The travis failures are irrelevant

**Comment by hmlnarik** [comment]
@vramik could you please rebase and fix KEYCLOAK issue number in the commit message?

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**